packages/clay-color-picker/demo/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
I think we should do the same here, to follow the pattern of the other components. Here's how I did it on [ClayDatePicker](https://github.com/liferay/clay/blob/9630c7157d9cb0b8c270a26353ceb630f0045c08/packages/clay-date-picker/demos/a11y.html).

It would also be interesting to continue creating the `a11y.html` page to follow the pattern.